### PR TITLE
Fix some mobile overflows

### PR DESCRIPTION
**Pull request by lexi-k**
I tried to fix at least some small things. Might attempt to fix more cosmetics later. Please note that I just edited the code on GitHub without any proper testing. (ofc I put the code in dev console to see if it at least works somehow)

### CODE REVIEW

File: src/pages/Home/Home.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const Home = ({ effectiveRole, instance = null, intl: { locale } }) => (
       <hr />
       <Row>
         <Col lg={12} xl={6}>
-          <Image src={`${URL_PATH_PREFIX}/public/logo.png`} className="text-center m-5" />
+          <Image src={`${URL_PATH_PREFIX}/public/logo.png`} className="text-center m-5 w-100 w-lg-auto" style={{ maxWidth: '66vw' }} />
```
[the remark this answers is not in the document]
Comment (lexi-k) [reply src/pages/Home/Home.js:40]: Thanks for the prompt reply! I would like to fix this (image attached).. Yeah I do not like the ad-hoc size neither, but do you have a better idea on how to solve it? 

image

[the remark this answers is not in the document]
Comment (lexi-k) [reply src/pages/Home/Home.js:40]: I can create proper css file for it, but I was not sure what is the preffered way of adding such css.

File: src/components/Users/UsersName/UsersName.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ const UsersName = ({
                 </a>
               )}
               {privateData && privateData.email && showEmail === 'full' && (
-                <small className="em-padding-left">
+                <Col className="em-padding-left d-inline d-md-block" style={{ float: window.innerWidth < 768 ? 'inline-start' : 'none' }}>
```
[the remark this answers is not in the document]
Comment (lexi-k) [reply src/components/Users/UsersName/UsersName.js:169]: I am not sure why I put `Col` there, to be honest, Copilot helped with inlining this one as I again did not want to create separate files, but that was apparently the wrong move. The float will fix this email causing overflow:

before:
image
after:
image

[the remark this answers is not in the document]
Comment (lexi-k) [reply src/components/Users/UsersName/UsersName.js:169]: But I just noticed I put the float on wrong emelent in this screenshot, the parentheses should also move according to the code in the PR itself. I can again move the code to a separate css file if you would like to fix those issues with css. I can come up with a better way of handling the css, the `float` was just a first simplest solution I came up with and I wanted to start a discussion first about the code style prefered (I should have mentioned it, but it was a long day for me so I could have communicated it better sorry).

File: src/pages/Home/Home.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (krulis-martin) [reply src/pages/Home/Home.js:40]: The first question we need to ask is "Should this be supported?" 1) The ReCodEx web app was never meant to support mobile devices (it is a very atypical situation). 2) The interface will look very bad on screens that are less than ~1000px wide anyway (the optimal width we aim for is FullHD), these two small patches will make little difference.
Furthermore, the UserName component is used in many places, it cannot be just patched for one special occurrence.

So, is there a "real" use case (a 390px-wide screen could be hardly considered real nowadays, not even on mobile devices) where a narrow screen causes issues that need to be fixed? If so, let's take a look at it and we shall find a better solution than the patches proposed in this PR. If not, I would respectfully advocate to dismiss these changes.

[the remark this answers is not in the document]
Comment (lexi-k) [reply src/pages/Home/Home.js:40]: I am using it on my mobile quite regurarly if someone asks me about homeworks and stuff. Tbh I thought that there is quite good mobile optimization in place already, just with some rough edges as not so many users use it on mobile presumably.

Well it buggs me for years at this point so I would really like to fix as much as I can for the mobile version in some reasonable time. Should I think of some *proper* solution to these problems and try to propose another PR (I will for sure close this "patches"), or should I pay you a visit IRL and we will disscuss it in person? Thanks!

[the remark this answers is not in the document]
Comment (krulis-martin) [reply src/pages/Home/Home.js:40]: A mobile app is currently not on the table, as there is no manpower to spare, but discussing this in person might help clarify the greatest issues of responsive design we can improve.

**Comment by lexi-k** [comment]
Closing as we need to discuss more offline to implement these changes properly.

**Comment by krulis-martin** [comment]
OK, contact me via Mattermost/Slack/mail...

**Pull request closed without merging**